src/migrations/m201207_065953_add_dataDeleted_column_archive_and_tax_categories.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace craft\commerce\migrations;
+
+use Craft;
+use craft\db\Migration;
+
+/**
+ * m201207_065953_add_dataDeleted_column_archive_and_tax_categories migration.
+ */
+class m201207_065953_add_dataDeleted_column_archive_and_tax_categories extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public function safeUp()
+    {
+        if (!$this->db->columnExists('{{%commerce_shippingcategories}}', 'pdfLanguage')) {
```

Comment:
I think the wrong column name is checked here.

Reply:
I fixed this on latest commit

Reply:
> Can we get some tests made for this feature? See also the other comments:

I created the ShippingCategoryTest with fixture under the unit test folder